validator/src/commands/wait_for_restart_window/mod.rs

```diff
@@ -0,0 +1,345 @@
+use {
+    crate::{admin_rpc_service, cli::DefaultArgs, new_spinner_progress_bar, println_name_value},
+    clap::{value_t_or_exit, App, Arg, ArgMatches, SubCommand},
+    console::style,
+    solana_clap_utils::{
+        input_parsers::pubkey_of,
+        input_validators::{is_parsable, is_pubkey_or_keypair, is_valid_percentage},
+    },
+    solana_rpc_client::rpc_client::RpcClient,
+    solana_rpc_client_api::config::RpcLeaderScheduleConfig,
+    solana_sdk::{
+        clock::{Slot, DEFAULT_S_PER_SLOT},
+        commitment_config::CommitmentConfig,
+        pubkey::Pubkey,
+    },
+    std::{
+        collections::VecDeque,
+        path::Path,
+        process::exit,
+        time::{Duration, SystemTime},
+    },
+};
+
+pub fn command(default_args: &DefaultArgs) -> App<'_, '_> {
```

Comment:
nit: I think all of these functions can just be `pub(crate)` right ?

Reply:
fixed: 918fd5ca6e